course_info/templates/course_info/partials/details.html

```diff
@@ -261,7 +261,6 @@ <h3 class="breadcrumbs">
                       <a href="{{site.course_site_url}}" target="_blank">{{site.course_site_url}}</a>
                     </div>
                     <div class="col-sm-3">
-                      <!-- todo: switch $index to site_id when available -->
                       <button type="button" class="btn btn-default pull-right"
                               ng-click="dc.dissociateSite($index)">
```

Comment:
Wouldn't it be better to use a map of sites than an index-based list, so that if multiple operations happened we wouldn't rely on the reliability of the indexes staying the same (whereas the `site_map_id` will be unique)?